test/index.ts

```diff
@@ -138,6 +138,33 @@ describe('.hasExpired()', () => {
   });
 });
 
+describe('.isTokenExpiring()', () => {
+  it('should exist', () => {
+    const gtoken = new GoogleToken();
+    assert.strictEqual(typeof gtoken.isTokenExpiring, 'function');
```

Comment:
I think we should add a test for the default behavior of `eagerRefreshThresholdMillis = 0`, which I believe should always indicate `isTokenExpiring()` of `false?

Reply:
I've added a test for this